### PR TITLE
DOCS-10887 - TPM=100

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Review by schinthakindims — COMMENTED** [review]
Since, in prod-eu the limit is 50, should we wait until we made it 100 in prod-eu as well?

@kimayap03  @podviaznikov

**Comment by kimayap03** [comment]
> Since, in prod-eu the limit is 50, should we wait until we made it 100 in prod-eu as well?
> 
> @kimayap03 @podviaznikov

We don't plan to change the TPM for prod-eu.

It is set here  https://github.com/mulesoft/kilonova-envs-config/blob/master/kprod-eu/access-management/access-management-authentication-server/values.yaml#L216

TPM number is probably different because of sizing. US has bigger DBs and more instances, they must measure values based on the usage. US is way more used than EU.

**Comment by hannanelson** [comment]
@schinthakindims - should I change the doc to reflect 50 in EU?

**Comment by schinthakindims** [comment]
> > Since, in prod-eu the limit is 50, should we wait until we made it 100 in prod-eu as well?
> > @kimayap03 @podviaznikov
> 
> We don't plan to change the TPM for prod-eu.
> 
> It is set here https://github.com/mulesoft/kilonova-envs-config/blob/master/kprod-eu/access-management/access-management-authentication-server/values.yaml#L216
> 
> TPM number is probably different because of sizing. US has bigger DBs and more instances, they must measure values based on the usage. US is way more used than EU.

@prafulrana Please verify if having different rate limit values  per env makes sense (100 in prod and 50 in prod-eu). I think it should be same.

**Comment by prafulrana** [comment]
i agree it should be the same.

**Comment by kimayap03** [comment]
Release docs only after https://www.mulesoft.org/jira/browse/MQ-2461 is fixed and deployed

**Review by schinthakindims — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/mq-apis.adoc [hunk 1]
```diff
@@ -52,6 +52,9 @@ The Anypoint MQ Admin API provides access to Anypoint MQ administrative function
 You can use the 
 https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-mq-admin/[Admin API] to create and manage apps, queues, and message exchanges.
 
+[NOTE]
+Anypoint MQ Admin API limits the number of transactions per minute (TPM) to 100 per caller ID (user or client ID).
```
Comment (kimayap03) [inline modules/ROOT/pages/mq-apis.adoc]: As per https://mulesoft.slack.com/archives/G0SLBAN7J/p1623799501032000?thread_ts=1623705371.025500&cid=G0SLBAN7J we wont be increasing limit in PROD-EU. 
So existing limits stay as is. prod limit is 100 and prod-eu is 50. 
Lets display lower limit to the customer.

`Anypoint MQ Admin API limits the number of transactions per minute (TPM) to 50 per caller ID (user or client ID).`

Comment (kimayap03) [reply modules/ROOT/pages/mq-apis.adoc:56]: @hannanelson ^

Comment (prafulrana) [reply modules/ROOT/pages/mq-apis.adoc]: How about "There  is a 100 transactions per minute (tpm) per client app limit to endpoints of the Admin API."

Comment (kimayap03) [reply modules/ROOT/pages/mq-apis.adoc]: @hannanelson Lets use the wording mentioned by praful, with a change in tpm to 50.

`There is a 50 transactions per minute (tpm) per client app limit to endpoints of the Admin API.`

Comment (hannanelson) [reply modules/ROOT/pages/mq-apis.adoc]: How about:
Client apps can send up to 50 transactions per minute (TPM) to Anypoint MQ Admin API endpoints.

Comment (hannanelson) [reply modules/ROOT/pages/mq-apis.adoc]: @prafulrana @kimayap03

File: modules/ROOT/pages/mq-apis.adoc [hunk 2]
```diff
@@ -52,6 +52,8 @@ The Anypoint MQ Admin API provides access to Anypoint MQ administrative function
 You can use the 
 https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-mq-admin/[Admin API] to create and manage apps, queues, and message exchanges.
 
+Client apps can sent up to 50 transactions per minute (TPM) to Anypoint MQ Admin API endpoints.
```
Comment (kimayap03) [inline modules/ROOT/pages/mq-apis.adoc]: typo: can `send`

Comment (kimayap03) [reply modules/ROOT/pages/mq-apis.adoc]: LGTM!

Comment (kimayap03) [reply modules/ROOT/pages/mq-apis.adoc]: Also we can change the PR title to reflect the value `TPM=50` for quick future ref

File: modules/ROOT/pages/mq-apis.adoc (hunk 1 above)
Comment (prafulrana) [reply modules/ROOT/pages/mq-apis.adoc]: "Your requests will be throttled to 50 tpm if you are calling the Admin API for distinct environments within a short time span." cc: @schinthakindims

File: modules/ROOT/pages/mq-apis.adoc (hunk 2 above)
Comment (hannanelson) [reply modules/ROOT/pages/mq-apis.adoc]: Dang, I thought I fixed that.

Comment (hannanelson) [reply modules/ROOT/pages/mq-apis.adoc]: The title of the PR has to have the Jira ticket number in it...

File: modules/ROOT/pages/mq-apis.adoc (hunk 1 above)
Comment (hannanelson) [reply modules/ROOT/pages/mq-apis.adoc]: @prafulrana - No. I didn't see that. I don't have access to the slack thread.

Comment (hannanelson) [reply modules/ROOT/pages/mq-apis.adoc]: @schinthakindims - what constitutes a "short time span"? Also, what does that mean "for distinct environments"?

Comment (schinthakindims) [reply modules/ROOT/pages/mq-apis.adoc]: @hannanelson , this limit only applies, if the calls to distinct environments are made. ie., 50 tpm to distinct environments. If it is the same environment, the information is cached and the limit doesn't apply.

Comment (kimayap03) [reply modules/ROOT/pages/mq-apis.adoc]: @hannanelson  Short timespan in this case is 1 minute

Comment (prafulrana) [reply modules/ROOT/pages/mq-apis.adoc]: Sounds good to me, what about you @schinthakindims we can use 1 min

**Review by schinthakindims — APPROVED** [review]
LGTM

File: modules/ROOT/pages/mq-apis.adoc [hunk 3]
```diff
@@ -52,6 +52,9 @@ The Anypoint MQ Admin API provides access to Anypoint MQ administrative function
 You can use the 
 https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-mq-admin/[Admin API] to create and manage apps, queues, and message exchanges.
 
+If you call the Anypoint MQ Admin API for distinct environments within one minute,
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/mq-apis.adoc:55]: Possible revision:
```suggestion
Requests to the Anypoint MQ Admin API are limited to 50 transactions per minute (TPM) per distinct environment.
```

**Review by IsaacEldridge — APPROVED** [review]

File: modules/ROOT/pages/mq-apis.adoc (hunk 3 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/mq-apis.adoc:55]: ooof, sorry @hannanelson The Anypoint MQ Admin API limits requests...

**Pull request merged**